doc/man3/OSSL_HTTP_transfer.pod

```diff
@@ -190,13 +190,20 @@ Since this function is typically called by applications such as
 L<openssl-s_client(1)> it uses the B<bio_err> and B<prog> parameters (unless
 NULL) to print additional diagnostic information in a user-oriented way.
 
-OSSL_HTTP_parse_url() parses its input string B<url> as a URL and splits it up
+OSSL_HTTP_parse_url() parses its input string B<url> as a URL
+of the form C<[http[s]://]address[:port][/path]> and splits it up
 into host, port and path components and a flag whether it begins with 'https'.
-The host component may be a DNS name or an IPv4 or an IPv6 address.
+The host component may be a DNS name or an IPv4 address
+or an IPv6 address enclosed in C<[> and C<]>.
 The port component is optional and defaults to "443" for HTTPS, else "80".
+If the B<pport_num> argument is NULL the port specification
+can be in mnemonic form such as "http" like with L<BIO_set_conn_port(3)>, else
+it must be in numerical form and its integer value is assigned to B<*pport_num>.
 The path component is also optional and defaults to "/".
-As far as the result pointer arguments are not NULL it assigns via
-them copies of the respective string components.
+As far as the result pointer arguments are not NULL the function assigns
+via them the respective url components.
+For string values these are non-NULL pointers to copies of the strings
+where B<*ppath> is guaranteed to begin with a C</> character.
```

Comment:
This 'where ... character' part should not be in the sentence.

Reply:
Changed this as well, see below.